bns/survey.js

```diff
@@ -129,8 +129,12 @@ upsert('WCSPROGRAMS_KoboBnsAnswer', 'AnswerId', {
   SubmissionUuid: dataValue('_uuid'),
   AnswerId: dataValue('_id'),
   LastUpdate: new Date().toISOString(),
-  SurveyDate: state =>
-    state.data.today ? state.data.today : state.data._submission_time,
+  SurveyDate: state => {
+    if (state.data.today.split('-')[0] >= 2014 ) {
```

Comment:
Here I believe today is a string so to compare it with `2014`you might need to do this: 
`if(Number(state.data.today.split('-')[0]) >= 2014)...`

Reply:
Ah got it, thanks!